website/content/docs/release-notes/v0_13_0.mdx

```diff
@@ -35,6 +35,17 @@ Boundary Enterprise has the same feature set as HCP Boundary and seamless migrat
 
 For more information, refer to [Boundary Enterprise](/boundary/docs/enterprise).
 
+<Note>
+
+The version of Go that was used in Boundary Enterprise release 0.13.0 contained CVE-2023-39326.
+Refer to the [advisory](https://github.com/advisories/GHSA-9f76-wg39-x86h) for more information.
+The issue was fixed in Go versions 1.21.5. Boundary was updated to use the new Go versions in Boundary Enterprise release 0.13.5.
+You should [upgrade Boundary](/boundary/tutorials/self-managed-deployment/upgrade-version) to version 0.13.5 or later.
+
+Community users should upgrade to version 0.14.1 or later.
```

Comment:
Quick note here that to use a community version with the fix, users will need to upgrade to 0.14.3, as 0.14.1 uses [go version 1.21.3](https://github.com/hashicorp/boundary/blob/release/0.14.1/.go-version)